### DISCUSSION TITLE
New working source: Stadtwerke Hürth (SWH)

### DISCUSSION THREAD

**Discussion by tmycgn** [mampfes/hacs_waste_collection_schedule#1703] [Ideas]
I have no idea where and how I can report new sources, but the integration of Stadtwerke Hürth (SWH) has been possible without any problems since this year (2024) via the following page.

https://www.stadtwerke-huerth.de/de/Muellkalender/Muellkalender.html

Simply right-click on the green button and "Copy link" after selecting the district and street. Then paste the url into the config and it works. :)

**Comment by 5ila5** [comment]
Thanks for the suggestion

Generally you can request sources in the issues tab: [Source Request](https://github.com/mampfes/hacs_waste_collection_schedule/issues/new?assignees=&labels=source+request&projects=&template=SOURCE-REQUEST.yml&title=%5BSource+Request%5D%3A+)

Documentation added by #1706

https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/ics/stadtwerke_huerth_de.md
